docs/admin/install/install.rst

```diff
@@ -2,24 +2,31 @@ Installing SecureDrop Workstation
 =================================
 .. include:: ../../includes/top-warning.rst
 
+.. warning::
```

Comment:
That makes sense. Since the timing is looking like the docs changes should land fairly closely with the packages, I think we are okay dropping the warning altogether. We can wait to tag the stable version until the packages are up, and if these changes are on `latest` for just a day or so I don't imagine it would cause anyone heartburn.